src/main.js

```diff
@@ -215,7 +215,7 @@ class MiniGraphCard extends LitElement {
     const name = this.tooltip.entity !== undefined
       ? this.computeName(this.tooltip.entity)
       : this.config.name || this.computeName(0);
-    const color = this.config.show.name_adaptive_color ? `opacity: 1; color: ${this.color};` : '';
```

Comment:
FYI all of these changes were not necessary as all these literals will be processed by minify-html-literals lib which you have added via rollup add-on